apps/openchallenges/organization-service/src/test/java/org/sagebionetworks/openchallenges/organization/service/OrganizationServiceApplicationTests.java

```diff
@@ -6,6 +6,7 @@
 @SpringBootTest
 class OrganizationServiceApplicationTests {
 
+  // integration test
```

Comment:
Are you sure that this is an integration test? Or this is also used to prepare the context for the unit tests?

Reply:
I'm not sure. That's the conclusion that I came to after looking into it. If it's used to prepare the context for the unit tests I will will change the comment.